### ISSUE TITLE
SByte gets serialized to string

### ISSUE THREAD

**Issue by dmytro-gokun** [ServiceStack/Issues#416]
This test:

```
        private class TestType1
        {
            public object Value { get; set; }
        }

        [Test]
        public void Serialize_SByteToObject_ShouldWork()
        {
            const string expected = @"{""Value"":100}";
            var obj = new TestType1 { Value = (sbyte)100 };

            var actual = JsonSerializer.SerializeToString(obj);
            actual.Should().Be(expected);
        }
```

fails with this message:

```
Expected string to be 
"{\"Value\":100}" with a length of 13, but 
"{\"Value\":\"100\"}" has a length of 15.
```

It looks like sbyte value gets serialized to a string for some reason.
I'm running 4.0.55

**Comment by mythz** [comment]
This should now be fixed from this commit: https://github.com/ServiceStack/ServiceStack.Text/commit/678dc0086f855f3b9eacdeb6c40212fb521dee84

This change is available from v4.0.55 that's now available on MyGet, since you already have v4.0.55 installed you'll need to [clear your NuGet packages cache](https://github.com/ServiceStack/ServiceStack/wiki/MyGet#redownloading-myget-packages) to download the latest v4.0.55 packages.

**Comment by dmytro-gokun** [comment]
Demis, thanks a lot for the quick fix!
